### ISSUE TITLE
CSS != SASS

### ISSUE THREAD

**Issue by martinheidegger** [consento-org/consento-website#42]
https://github.com/consento-org/consento-website/blob/83388cf4ed6370dab396b7de1469d0fb31595ac9/home.hbs#L204 indicates that this is a sass style, but afaik its css, right? Also: why not put the css code together in the stylesheet?

https://github.com/consento-org/consento-website/blob/master/assets/main/sass/main.scss

**Comment by Torgian** [comment]
Yes, that was a mistake from earlier; I know this is css.

As for putting it into a single css style sheet, this is something I've been wanting to do for a while, but thought keeping things separated between files would be easier to debug until most of it was ready. 

That said, I've decided to try and move things into an scss style sheet. Unfortunately, I can't seem to get gulp to see / use the main.scss file correctly. I have a test style sheet ready for home.hbs, but I think the gulp config file isn't configured correctly. 

I'll work on this more tomorrow night, but if I cannot figure it out, I will simply start making a new .css file instead.

**Comment by Torgian** [comment]
I was able to get Sass / SCSS to compile correctly locally using gulp, so I am working on migrating the css to sass now.

**Comment by Torgian** [comment]
Refer to commit [eaa3cbc](https://github.com/consento-org/consento-website/commit/eaa3cbc7bb06f139f341009c2811402ec7e0f8a4) for changes. This will most likely affect past fixes.

**Comment by martinheidegger** [comment]
So I checked the website and I fear that quite a lot of things have broken majorly, which turns this commit into a few steps backwards. This is an issue that often appears when one doesn't have a good testing framework. There is a methodology to to make sure that only very few issues sneak in. Please revert the code for now and let's do this once more in a pair programming session

**Comment by Torgian** [comment]
Reverted, see commit [5d2d0214](https://github.com/consento-org/consento-website/commit/5d2d02140ab904383c5ad3b860c5d115c348342e)

**Comment by Torgian** [comment]
I have pulled fresh from the `next` branch after the revert, and have also installed all required packages. Ran with `npm run dev`. 

I changed the files over one by one, into SASS, and removed styling within the `topbar.hbs, navigation.hbs, home.hbs, footer.hbs, default.hbs` pages. I could not duplicate what was previously happening on the live server page (jumbled up menu, etc)

Here are some screenshots. 

![localhost_2368_(iPhone 6_7_8)](https://user-images.githubusercontent.com/8596793/74145516-fc6bee00-4c39-11ea-9952-b9f08aa709cc.png)
![localhost_2368_(iPad Pro)](https://user-images.githubusercontent.com/8596793/74145522-ffff7500-4c39-11ea-926f-f36c6d440c0b.png)
